### DISCUSSION TITLE
Codeigniter 3.1.13

### DISCUSSION THREAD

**Discussion by gyveryeoh** [bcit-ci/CodeIgniter#6144] [General]
Just want to know if codeigniter3.1.13 is still usefull until now?.. im using vps with 7.4 version.
I am still maintaning almost 5-8 systems with C.I3 version. im afraid if one day my clients encountered an errors or glitches while using the system with C.I 3 version.

**Comment by khashabawy** [comment]
Yes it is useful , for at least 20 years it will be valid

Wordpress websites still running php 5
